integration/app/test-demo/server.js

```diff
@@ -0,0 +1,165 @@
+/* eslint-disable */
```

Comment:
This looks a lot like the main demo - even though that is in a different repo. Is it possible to move this stuff into that repo instead?

Reply:
Yeah it is the same one, I copied from main demo.
We don't have such a server.js file in React Component Library. If we move it into that repo, how should we enable the backend for this demo?

Reply:
Well the whole point of having these tests is to run them as CI Workflow right? Does the intergation tests needt o be part of this repo or can we have it in the other repo, and then in our github action we just check out that repo and run the integ tests?

Reply:
Based on last week's discussion, we will keep all integration test related code in this repo.